### PR TITLE
Address missing test coverage indicated in Simplecov

### PR DESCRIPTION
**Pull request by ggerdsen**
Add tests to ensure coverage and remove unused methods, comments and whitespace.

### CODE REVIEW

File: app/controllers/merchant/orders_controller.rb [hunk 1]
```diff
@@ -4,14 +4,4 @@ def show
     @order = Order.find(params[:order_id])
   end
```
Comment (sienna-kopf) [inline app/controllers/merchant/orders_controller.rb:6]: Oops about the duplicate methods. Nice fix here

File: spec/features/items/merchant_items_index_spec.rb [hunk 2]
```diff
@@ -39,5 +40,90 @@
...
+      end
+
+      expect(current_path).to eq("/merchant")
+
+      click_on "My Items"
+      
+      expect(current_path).to eq("/merchant/items")
+      expect(page).to have_content("#{@tire.name}")
+      
+      click_on("delete", match: :first)
```
Comment (sienna-kopf) [inline spec/features/items/merchant_items_index_spec.rb:62]: Cool syntax here!

File: spec/models/merchant_spec.rb [hunk 3]
```diff
@@ -21,10 +21,10 @@
     end
     it 'no_orders' do
       expect(@meg.no_orders?).to eq(true)
-
+      
       order_1 = Order.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033, user_id: @user.id)
       item_order_1 = order_1.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
-
+
```
Comment (sienna-kopf) [inline spec/models/merchant_spec.rb:27]: Awesome job writing really thorough tests on this Garrett
